app/(home)/page.tsx

```diff
@@ -70,6 +72,46 @@ export default function HomePage() {
             </Link>
           </div>
         </div>
+
+        {/* Social Media Links */}
+        <div className="absolute bottom-16 left-0 right-0 flex justify-center gap-8 z-10">
+          <Link 
+            href="https://x.com/TheEtherGuild" 
+            target="_blank" 
+            rel="noopener noreferrer"
+            className="text-white hover:text-gray-300 transition-colors"
+            aria-label="X (Twitter)"
+          >
+            <FaXTwitter size={28} />
+          </Link>
+          <Link 
+            href="https://discord.gg/sdCzcnMHVb" 
```

Comment:
```suggestion
            href="https://discord.com/invite/sdCzcnMHVb" 
```
this one works for me